glusterfsd/src/glusterfsd.h

```diff
@@ -116,6 +116,7 @@ enum argp_option_keys {
     ARGP_BRICK_MUX_KEY = 193,
     ARGP_FUSE_DEV_EPERM_RATELIMIT_NS_KEY = 194,
     ARGP_FUSE_INVALIDATE_LIMIT_KEY = 195,
+    ARGP_START_MODE_KEY = 'm',
```

Comment:
Why 'm' ? shouldn't it be a number as the previous ones ?

Reply:
Nothing specific as such, since this was a type of start **mode** so thought of using `m` for it.